clients/python-wrapper/lakefs/config.py

```diff
@@ -68,20 +69,27 @@ def __init__(self, verify_ssl: Optional[bool] = None, proxy: Optional[str] = Non
                 self.server = ClientConfig.Server(**data["server"])
                 self.credentials = ClientConfig.Credentials(**data["credentials"])
             found = True
+        except KeyError:	# in case credentials or server is not elaborated
+            pass
         except FileNotFoundError:  # File not found, fallback to env variables
             self.server = ClientConfig.Server(endpoint_url="")
             self.credentials = ClientConfig.Credentials(access_key_id="", secret_access_key="")
 
         endpoint_env = os.getenv(_LAKECTL_ENDPOINT_ENV)
         key_env = os.getenv(_LAKECTL_ACCESS_KEY_ID_ENV)
         secret_env = os.getenv(_LAKECTL_SECRET_ACCESS_KEY_ENV)
+        access_token_env = os.getenv(_SESSION_TOKEN)
 
         self.host = endpoint_env if endpoint_env is not None else self.server.endpoint_url
         self.username = key_env if key_env is not None else self.credentials.access_key_id
         self.password = secret_env if secret_env is not None else self.credentials.secret_access_key
+        self.access_token = access_token_env
+
+        if self.access_token is not None and len(self.host) > 0:
```

Comment:
Why is the check dependant on len of self.host?
How is it any different from the statement below?